js/components/widgets/thumbnailer.vue

```diff
@@ -167,8 +165,8 @@ export default {
                     boxWidth: max_width,
                     boxHeight: max_height
                 }, function() {
```

Comment:
Not in this case, the arrow shouldn't be used because the `jCrop` only expose the instance through the `this` scope and we need it (see the `that.jCrop = this`, this is the only way to get it or at least documented in the jCrop documentation)